src/code_style.md

```diff
@@ -0,0 +1,37 @@
+# Clear Code  
+
+1. Code should be clear and concise.  
+2. Variables in the code should have meaningful names.  
+3. Function naming should follow established theories and naming conventions.  
+4. Avoid functions longer than 50 lines to improve code readability. 
```

Comment:
I dont understand, a function should not have more than 50 lines **of code**? Is that what you mean?